tests/06-hello-font/input/amitu/index.ftd

```diff
@@ -9,4 +9,26 @@ font: $assets.fonts.Roboto
 font: $assets.fonts.myFirstFont
 
 -- ftd.text: World
-font: $theme-assets.fonts.themeFont
\ No newline at end of file
+font: $theme-assets.fonts.themeFont
```

Comment:
@amitu , check this for the usage. I've namespaced all the files within a `files` record. This way if someone has a folder named `font` it won't conflict.